provider/inspec.rb

```diff
@@ -16,6 +16,7 @@
 require 'provider/inspec/manifest'
 require 'provider/inspec/resource_override'
 require 'provider/inspec/property_override'
+require 'active_support/inflector'
```

Comment:
When this gets loaded in, it loads in a whole bunch of methods and adds them to the `::String` class. Don't be surprised if this causes some strange behavior elsewhere.